FordLabs/PeopleMover
# 104 skip to next product

## Issue
Resolves 104

## What was done
- [x] Added a hidden anchor tag that is only visible when focused (same as the "skip to main content" button)
- [x] Added IDs on product card containers for the above anchor tags to reference
- [x] De-duplicated the rendering logic for the product card sections

## How to test
1. Go to a space
2. Using only keyboard, navigate to the first product
3. Observe the new "Jump to next product" button appeared as the first element before the actual product card
4. Click "enter"
5. Hit "tab"
6. Note that your focus is now on the next product
7. Continue skipping to next product
8. Note that the last product does not have a "skip to next product" option
9. Sort by location or product tag
10. Note that each grouping / section has the "skip to next product" button except for the last product in that section

## Accessiblity Criteria


## Notes

### For Designers
- The VoiceOver experience jumping from product to product doesn't announce which product we're on or which product we're going to - what would you like to see in this regard?

### For Developers
- In the implementation of this story, we've encountered an interesting divergence between functionality in Chrome and Firefox - when we put the focus onto an unfocusable element in Chrome then hit tab, it correctly focuses the first focusable element within that unfocusable element. When in Firefox however, we see it focus the _second_ focusable element within that unfocusable element. Thoughts?
